haas/client/auth.py

```diff
@@ -0,0 +1,13 @@
+import base64
+
+
+
+def auth_db(username, password):
```

Comment:
There's no need to wait for #615; the `HTTPClient` stuff is already merged (via #626).